presto-main/src/test/java/com/facebook/presto/util/TestTimeZoneUtils.java

```diff
@@ -67,6 +67,13 @@ public void testNamedZones()
                 continue;
             }
 
+            if (zoneId.equals("Europe/")) {
+                // TODO: Remove once minimum Java version is increased 11.0.10
```

Comment:
When the Joda version was upgraded in this PR: https://github.com/prestodb/presto/pull/19758, the release notes mentioned that the runtime version needed to be upgraded to have the relevant timezone data. 

If we have to exclude, we should not exclude all Europe timezones from tests but only the ones which were added as part of the upgrade. 

Also, did you also try running the test using the mentioned Java runtime version (11.0.10)?

Reply:
![image](https://github.com/prestodb/presto/assets/89628774/8af10202-8569-4621-abbe-3d46bf200af6)

Sorry its 17 no error i have corrected accordingly, with 
![image](https://github.com/prestodb/presto/assets/89628774/45890437-21be-490f-8dee-8bb6f72092cd)
 same issue is coming with java 11

Reply:
![image](https://github.com/prestodb/presto/assets/89628774/309e444b-40fb-4a0f-a6d4-046671c07fc4)

Reply:
openjdk version "17.0.7" 2023-04-18
OpenJDK Runtime Environment Homebrew (build 17.0.7+0)
OpenJDK 64-Bit Server VM Homebrew (build 17.0.7+0, mixed mode, sharing)
 this version of java , no issue is coming

Reply:
@imjalpreet please help to review this PR.

Reply:
@tdcmeehan please help to review the PR.